src/deepakBorse/Assignments/Test3_midcharMaxlen.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Test - 3 : 10th Sep'2022
+
+Program 1 : (20 mins)
+Return middle character of max length String from given array.
+input : {"Krishna","Maulik","Aashvi","Technocredits","Harsh"}
+output : c
+
+Program 2: (40 mins)
+Write a method to return middle character of each elements from given array.
+input : {"Mahesh","Avni","Puja","Harshal","Aditty","Rahul"}
+output :      Mahesh -> h
+              Avni -> v
+              Puja -> u
+              Harshal -> s
+              Aditty -> i
+              Rahul -> h
+ */
+package deepakBorse.Assignments;
+
+import java.util.Arrays;
+
+public class Test3_midcharMaxlen {
+
+	char maxchaMaxlen(String[] namearry) {
+		int maxlen = namearry[0].length();
+		String maxlenname = namearry[0];
+		int index1 = 0;
+
+		for (int i = 1; i < namearry.length; i++) {
```

Comment:
good to see that you have started loop from index 1, as you have already taken max value as 0